src/less/variables.less

```diff
@@ -33,6 +33,8 @@
 @global-color-warning: #f2c746;
 @global-color-warning-light: tint(@global-color-warning, 92%);
 @global-color-emphasis: #fdf9c7;
+@global-color-old-value: #dbedb2;
+@global-color-new-value: #f3aeb2;
```

Comment:
variable name should generic and it should specify the color, not that where we are going to use this variable.

Reply:
Done @mahimaag